keyboards/handwired/dactyl_rah/config.h

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 Ralph Azucena <ralphie02@live.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x3060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    ralphie02
+#define PRODUCT         DACTYL_RAH
+#define DESCRIPTION     An ortholinear, split, 3D-curved keyboard with thumb clusters.
```

Comment:
```suggestion
```

Reply:
Why does this have to be removed?? The directory I forked this from defined this value, although, it is pretty old so I wouldn't be surprised if it's deprecated.

Reply:
Honestly, I'm not sure that it was ever actually used by the USB endpoints.  So ... it doesn't do anything.  So we've been removing it, and flagging it on new boards.

Reply:
ok, good enough reason for me 👍